.github/workflows/publish.yml

```diff
@@ -28,4 +28,4 @@ jobs:
               --repo="$GITHUB_REPOSITORY" \
               --title="${{github.ref_name}}" \
               --generate-notes \
-              build/script.elementum.jackett.zip
```

Comment:
I'm not a big fan of this to be honest. The version is already in the GitHub URL when downloading. Is there a good reason for this?

Reply:
yes, downloaded zip need to be different. I can't rollback the version because i will overwrite the old one, with the same name file. 
For example. Any *.rpm package hase a version in name, while it also in repo and inside the package itself.

Reply:
What do you mean you can't rollback the version? The version is inside the `addon.xml` anyway?

Reply:
I'm not sure why I wrote "can't". I can, but I need a reason. 
I explained my reasoning:
1. World best practice. It's crucial for me.
2. Ability to store multiple versions on disc, no need to rename. 
3. If I remember correctly, @elgatito told it's crucial for storing artifacts in Kodi repository.

I didn't understand your reasons.